arch/arm/src/nrf52/nrf52_gpiote.c

```diff
@@ -135,6 +141,61 @@ static int nrf52_gpiote_isr(int irq, FAR void *context, FAR void *arg)
         }
     }
 
+  /* Check for PORT event */
```

Comment:
I think this should be configurable with Kconfig. There is no point in including this logic if we don't want to use SENSE mechanism

Reply:
If we do so, we should disable the set_pin_event function as well, so that if one wants to use this mechanism it can depend on the corresponding config symbol. But as I said before, it may be better to always support it (once I make the change to only scan the set bits in LATCH).